### ISSUE TITLE
Shorten works incorrectly after upgrade to ownCloud 8

### ISSUE THREAD

**Issue by noisem** [Fmstrat/shorten#7]
instead of short link it creates an html text

**Comment by noisem** [comment]
I've reinstalled shorten in ownCloud 8 and now it just not generate the short link. In previous installation I've changed setTimeout(addShareListener, 3000); in  js/scripts.js, this is why it generated html text instead of short link. And even if there is short link in db, when it try to open  code.php:
for ex. https://owncloud.example.com/index.php/apps/shorten/code.php?code=j2IjyK, then it even doesn't enter in code.php but redirect to authentication page right away,
